packages/backend/src/migration/1593764167709-alterTables.ts

```diff
@@ -8,14 +8,18 @@ export class alterTables1593764167709 implements MigrationInterface {
         await queryRunner.query(`DROP SEQUENCE "modos"."observation_id_seq";`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "id" ADD GENERATED ALWAYS AS IDENTITY;`, undefined);
 
-        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "position" geometry(Point, 4326) NOT NULL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "position" geometry(Point, 4326);`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."observation" ADD COLUMN "in_range" BOOLEAN NOT NULL DEFAULT FALSE;`, undefined);
```

Comment:
If we got a mix of camel and snake case, it the begining of problem. Stick with CamelCase.

Reply:
Outdated.